### DISCUSSION TITLE
Idea: what if you make a weblate?

### DISCUSSION THREAD

**Discussion by LNLenost** [tfdsoft/famidash#158] [Ideas]
Maybe you guys can add a menu to choose from 2 or 3 languages

**Comment by glacierjl** [comment]
Translating Famidash would take up more effort and (admittedly sparse and crucially needed) ROM space than it's honestly worth it. Those who want to translate it themselves are free to do so by forking the project, though.

**Reply by LNLenost** [reply]
Right
